GRDB/Record/PersistableRecord+Encodable.swift

```diff
@@ -194,6 +214,92 @@ private struct PersistableRecordEncoder : Encoder {
+    func encode(_ value: Int16, forKey key: KeyType) throws { throw errorMessage }
+    func encode(_ value: Int32, forKey key: KeyType) throws { throw errorMessage }
+    func encode(_ value: Int64, forKey key: KeyType) throws { throw errorMessage }
+    func encode(_ value: UInt, forKey key: KeyType) throws { throw errorMessage }
+    func encode(_ value: UInt8, forKey key: KeyType) throws { throw errorMessage }
+    func encode(_ value: UInt16, forKey key: KeyType) throws { throw errorMessage }
+    func encode(_ value: UInt32, forKey key: KeyType) throws { throw errorMessage }
+    func encode(_ value: UInt64, forKey key: KeyType) throws { throw errorMessage }
+    func encode(_ value: Float, forKey key: KeyType) throws { throw errorMessage }
+    func encode(_ value: Double, forKey key: KeyType) throws { throw errorMessage }
+    func encode(_ value: String, forKey key: KeyType) throws { throw errorMessage }
+    func encode<T>(_ value: T, forKey key: KeyType) throws where T : Encodable { throw errorMessage }
+
+    func nestedContainer<NestedKey>(keyedBy keyType: NestedKey.Type, forKey key: KeyType) -> KeyedEncodingContainer<NestedKey> where NestedKey : CodingKey {
+        fatalError("Not implemented")
+    }
+    func nestedUnkeyedContainer(forKey key: KeyType) -> UnkeyedEncodingContainer {
+        fatalError("Not implemented")
+    }
+    func superEncoder() -> Encoder {
+        fatalError("Not implemented")
+    }
+    
+    func superEncoder(forKey key: KeyType) -> Encoder {
+        fatalError("Not implemented")
+    }
+}
+
+class ThrowingUnkeyedContainer: UnkeyedEncodingContainer {
+    let errorMessage: Error
+    var codingPath: [CodingKey] = []
+    var count: Int = 0
+    
+    init(error: Error) {
+        errorMessage = error
+    }
+    
+    func encode(_ value: Int) throws { throw errorMessage }
+    func encode(_ value: Int8) throws { throw errorMessage }
+    func encode(_ value: Int16) throws { throw errorMessage }
+    func encode(_ value: Int32) throws { throw errorMessage }
+    func encode(_ value: Int64) throws { throw errorMessage }
+    func encode(_ value: UInt) throws { throw errorMessage }
+    func encode(_ value: UInt8) throws { throw errorMessage }
+    func encode(_ value: UInt16) throws { throw errorMessage }
+    func encode(_ value: UInt32) throws { throw errorMessage }
+    func encode(_ value: UInt64) throws { throw errorMessage }
+    func encode(_ value: Float) throws { throw errorMessage }
+    func encode(_ value: Double) throws { throw errorMessage }
+    func encode(_ value: String) throws { throw errorMessage }
+    func encode<T>(_ value: T) throws where T : Encodable { throw errorMessage }
+    func encode(_ value: Bool) throws { throw errorMessage }
+    func encodeNil() throws { throw errorMessage }
+    
+    func nestedContainer<NestedKey>(keyedBy keyType: NestedKey.Type) -> KeyedEncodingContainer<NestedKey> where NestedKey : CodingKey {
+        fatalError("Not implemented")
```

Comment:
Same comment as above: there should be no fatalError in this type.

Reply:
commit c52b59f1 also returns another throwing container here